reactor-core/src/main/java/reactor/core/publisher/FluxSubscribeOnCallable.java

```diff
@@ -270,7 +277,17 @@ public void request(long n) {
 					if (s == NO_REQUEST_HAS_VALUE) {
 						if (STATE.compareAndSet(this, s, HAS_REQUEST_HAS_VALUE)) {
 							try {
-								Disposable f = scheduler.schedule(this::emitValue);
+								Disposable f = scheduler.schedule(new Scheduler.ContextRunnable() {
```

Comment:
Note: this creates greater GC pressure compared to a method reference, but we might have to live with it. Fortunately it is not on the critical path, as a CallableSubscription is most likely only requested only once.

Reply:
actually not that much of a difference because it is a capturing method reference on `this`, not a static method. bytecode-wise this would still avoid creation of a synthetic method, but that's another topic